Libraries/Hotcakes.Payment/packages.config

```diff
@@ -1,4 +1,9 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
   <package id="AuthorizeNet" version="1.8.2" targetFramework="net40" />
+  <package id="Microsoft.Bcl.AsyncInterfaces" version="1.1.0" targetFramework="net472" />
+  <package id="Newtonsoft.Json" version="9.0.1" targetFramework="net472" />
```

Comment:
There are multiple references to `Newtonsoft.Json` in this update, and the versions don't match.  We need to prevent compile-time warnings and potential exceptions/debugging issues later.  Please be sure all identical references reflect the same reference version.

Reply:
Fixed